CHANGELOG

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [3.8.0] - 2023-06-13
```

Comment:
I created a new development/3.9 branch and redirected your pr to it, as 3.8 has already been released and we're adding some feature into bert-e

```suggestion
## [3.9.0] - 2023-07-20
```